README.md

```diff
@@ -47,12 +47,12 @@ This provider's versions are compatible with the following versions of Cluster A
 
 This provider's versions are able to install and manage the following versions of Kubernetes:
 
-|                                    | v1.16 | v1.17 | v1.18 | v1.19 | v1.20 | v1.21 |
-| ---------------------------------- | ----- | ----- | ----- | ----- | ----- | ----- |
-| OpenStack Provider v1alpha3 (v0.3) | ✓    | ✓    | ✓     | ✓    | ✓    |       |
-| OpenStack Provider v1alpha4 (v0.4) |       |       |       |       | ✓    | ✓     |
-| OpenStack Provider v1alpha4 (v0.5) |       |       |       |       | ✓    | ✓     |
-| OpenStack Provider v1alpha5        |       |       |       |       |       | ✓     |
+|                                    | v1.16 | v1.17 | v1.18 | v1.19 | v1.20 | v1.21 | v1.22 | v1.23 |
+| ---------------------------------- | ----- | ----- | ----- | ----- | ----- | ----- | ----- | ----- |
+| OpenStack Provider v1alpha3 (v0.3) | ✓     | ✓     | ✓     | ✓     | ✓     |       |       |       |
+| OpenStack Provider v1alpha4 (v0.4) |       |       |       |       | ✓     | ✓     | ✓     | ✓     |
```

Comment:
do we have such test for `OpenStack Provider v1alpha4 (v0.4)` + v1.22/v1.23?

Reply:
You're right 👍  I removed this ones. I also switched the v1.22 ones to `+` (`should work, but we weren't able to test it`)

I've got no strong opinions what to note exactly.

This PR introduces the testing for v1.23.